app/Http/Controllers/MasjidProfileController.php

```diff
@@ -39,4 +40,32 @@ public function update(Request $request): RedirectResponse
 
         return redirect()->route('masjid_profile.show');
     }
+
+    public function image(Request $request)
+    {
+        $this->authorize('edit_masjid_profile');
+
+        $validatedPayload = $request->validate([
+            'image' => 'required'
+        ]);
+
+        if (!base64_decode($validatedPayload['image'])) {
+            return response()->json([
+                'message' => 'Image not found'
+            ]);
+        }
+
+        $image_parts      = explode(";base64,", $request->image);
```

Comment:
Beberapa catatan:
1. Nama variable kita pakai camelCase semua
2. Jarak antara syntax dengan tanda sama dengan cukup 1 spasi ` = `
3. Usahakan selalu pakai tanda petik 1 (`'`), tanda petik 2 hanya dipakai kondisi tertentu saja.
4. Setelah proses validasi, jangan pakai `$request` lagi untuk memanggil data request, kita pakai `$validatedPayload`

```suggestion
        $imageParts = explode(';base64,, $validatedPayload['image']);
```